tests/unit/store/playback.spec.js

```diff
@@ -84,13 +83,13 @@ describe("store/playback", () => {
 
       store.commit("playback/stop_playback_progression");
 
-      expect(spied_clear_interval).not.toBeCalled();
+      expect(spied_clear_interval).not.toHaveBeenCalled();
     });
-    test("Given tooltips_delay is having a default value of 2 sec, then modified to 1 sec, the Vuex state is updated", () => {
+    test("Given tooltips_delay is having a default value of 2 sec, When modified to 1 sec, Then the Vuex state is updated", () => {
```

Comment:
try not to include specific numbers in the test title. We want the title to generically describe the concept being tested. For example, `When a value different than the default value is committed to tooltips_delay, Then the Vuex state is updated`